database/seeders/FrontendSeeder.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Database\Seeders;
+
+use Blumilk\Meetup\Core\Models\Meetup;
+use Blumilk\Meetup\Core\Models\Organization;
+use Blumilk\Meetup\Core\Models\Speaker;
+use Blumilk\Meetup\Core\Models\User;
+use Illuminate\Database\Seeder;
+use Illuminate\Support\Carbon;
+use Illuminate\Support\Facades\Hash;
+
+class FrontendSeeder extends Seeder
```

Comment:
```suggestion
class DummyDataSeeder extends Seeder
```
that will be more readable